.circleci/config.yml

```diff
@@ -52,42 +81,28 @@ jobs:
     steps:
       - buildevents/with_job_span:
           steps:
+            - buildevents/start_trace
             - checkout
-            - buildevents/berun:
-                bename: build
-                becommand: make
-            - buildevents/berun:
-                bename: lint
-                becommand: make lint
-            # - buildevents/berun:
-            #     bename: test
-            #     becommand: make test
+            - build
+            - lint
+            # - test
             - save_go_cache
             - persist_to_workspace:
                 root: .
                 paths: [ . ]
-
   release:
     executor: go
     steps:
       - buildevents/with_job_span:
           steps:
+            - install_ghr
+            - aws-cli/install
+            - aws-cli/setup
```

Comment:
does this step need the AWS secrets (aws-access-key-id, etc)?

Reply:
It does! Despite the brevity here, the step will get them.

This step in the `release` job gets the secrets it needs from the context passed to the job when [called during the `ci` workflow](https://github.com/honeycombio/influx2hny/pull/5/files/a46fc952fad9687fa89e20bbdb6d31304a6cf66b#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R124). I had tried declaring the context in the release job definition, but Circle barked about that being invalid. The [awscli orb](https://circleci.com/developer/orbs/orb/circleci/aws-cli?version=1.4.0)'s `setup` step defaults a bunch of its parameters to the usual AWS environment variables. We have those set in our context.

Reply:
Makes sense, cool!